packages/components/src/FormField/FormField.tsx

```diff
@@ -167,6 +184,17 @@ export function FormField(props: FormFieldProps) {
           onChange && onChange(newValue);
           onControllerChange(event);
         }
+        function handleOnValueChange(values: {
```

Comment:
Maybe we should called this `maskedOnValueChangedHandler` to make it clear it's related to the masking code.

Reply:
@andrewjdmccann I am leaving this one to your discretion :)